src/modules/70-pipeline/components/PipelineStudio/RightBar/RightBarUtils.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+export const isRuntimeInput = (str: any): boolean => typeof str === 'string' && str?.includes('<+input>')
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232315)
```suggestion
export const isRuntimeInput = (str: unknown): boolean => typeof str === 'string' && str?.includes('<+input>')
```